iina/KeyBindingTranslator.swift

```diff
@@ -7,9 +7,106 @@
 //
 
 import Foundation
+import Mustache
 
 class KeyBindingTranslator {
 
+  static let l10nDic: [String: String] = {
+    let filePath = Bundle.main.path(forResource: "KeyBinding", ofType: "strings")!
+    let dic = NSDictionary(contentsOfFile: filePath) as! [String : String]
+    return dic
+  }()
+
+  static let mpvCmdFormat: [String: String] = {
+    let filePath = Bundle.main.path(forResource: "MPVCommandFormat", ofType: "strings")!
+    let dic = NSDictionary(contentsOfFile: filePath) as! [String : String]
+    return dic
+  }()
+
+  static private let UnsupportedCmdPrefix: [String] = [
+    "no-osd", "osd-auto", "osd-bar", "osd-msg-bar", "raw", "repeatable", "expand-properties"
+  ]
+
+  static func readable(fromCommand action: [String]) -> String {
```

Comment:
I'd recommend renaming this function to something that gives a bit more insight about its parameters and return value. If I'm not mistaken, it converts an action into a readable command, so something like `readableCommand(from action: [String])` might be a better choice.